distributed/protocol/compression.py

```diff
@@ -93,6 +93,41 @@ def _fixed_lz4_decompress(data):
     }
     default_compression = "lz4"
 
+with ignoring(ImportError):
+    _import_policy = dask.config.get("distributed.comm.zstd.import_policy", "default")
+    os.environ["PYTHON_ZSTANDARD_IMPORT_POLICY"] = _import_policy
+
+    import zstandard as zstd
+
+    zstd_level = dask.config.get("distributed.comm.zstd.level", None)
+    zstd_kwargs = dict(
+        write_checksum=dask.config.get("distributed.comm.zstd.write_checksum", None),
+        write_content_size=dask.config.get(
+            "distributed.comm.zstd.write_content_size", True
+        ),
+        write_dict_id=dask.config.get("distributed.comm.zstd.write_dict_id", True),
+        threads=dask.config.get("distributed.comm.zstd.threads", 0),
+    )
```

Comment:
Can you add the default configuration values to this file?  https://github.com/dask/distributed/blob/master/distributed/distributed.yaml

Then you can remove these default values from the code here.